### PR TITLE
updated german translation for 2.2.

### PR DESCRIPTION
**Pull request by sluetze**
Signed-off-by: sluetze <13255307+sluetze@users.noreply.github.com>

may be WIP, if more wordings in 2.2 are changed.

### CODE REVIEW

File: src/portal/src/i18n/lang/de-de-lang.json [hunk 1]
```diff
@@ -1595,71 +1595,71 @@
...
-        "EXPIRES_AT": "Expires at",
-        "VIEW_SECRET": "REFRESH SECRET",
+        "NEW_TOKEN": "Neues Secret",
+        "REFRESH": "AKTUALISIEREN",
+        "PROJECTS_MODAL_TITLE": "Projekte des Robot-Accounts",
+        "PROJECTS_MODAL_SUMMARY": "Diese Projekte werden durch den Robot-Account genutzt.",
+        "CREATE_ROBOT": "Systemweiten Robot-Account anlegen",
+        "CREATE_ROBOT_SUMMARY": "Einen systemweiten Robot-Account anlegen, der auf einzelne Projekte berechtigt wird. Wähle \"Alle Projekte abdecken\", um den Robot-Account für alle existierenden und zukünftigen Projekte zuzulassen",
+        "EDIT_ROBOT": "Systemweiten Robot-Account bearbeiten",
+        "EDIT_ROBOT_SUMMARY": "Bearbeite einen systemweiten Robot-Account. Wähle \"Alle Projekte abdecken\", um den Robot-Accoutn für alle existierenden und zukünftigen Projekte zuzulassen",
```
Comment (Vad1mo) [inline src/portal/src/i18n/lang/de-de-lang.json:1631]: Bearbeite ein System Robot Account. Wähle \"Alle Projekte abdecken\", um auf alle bestehenden und zukünftigen Projekte angewendet zu werden.

**Review by Vad1mo — CHANGES_REQUESTED** [review]
The translation is a bit stiff. There are a few English words still in use although good german word exist also in the IT lingua. 

Roboter-Account -> Roboterkonto
Secret -> Geheimnis

File: src/portal/src/i18n/lang/de-de-lang.json [hunk 2]
```diff
@@ -1595,71 +1595,71 @@
...
-        "ENABLE_ROBOT_SUCCESSFULLY": "Enabled robot successfully",
-        "DISABLE_ROBOT_SUCCESSFULLY": "Disabled robot successfully",
-        "ROBOT_ACCOUNT": "Robot account",
-        "PROJECTS": "Projects",
-        "ALL_PROJECTS": "All projects with",
-        "PERMISSIONS": "PERMISSION(S)",
-        "REFRESH_SECRET_SUMMARY": "Refresh the robot account secret automatically, or optionally open the details to manually specify a new secret",
+        "ADD_ROBOT": "Robot hinzufügen",
+        "UPDATE_ROBOT": "Robot aktualisieren",
+        "UPDATE_ROBOT_SUCCESSFULLY": "Robot aktualisieren erfolgreich",
```
[the remark this answers is not in the document]
Comment (sluetze) [reply src/portal/src/i18n/lang/de-de-lang.json:1605]: yes, that's cleaner. I chose the "<Subject> <Verb> erfolgreich" and "<Subject> <Verb> fehlgeschlagen" on purpose. The idea was, that you only have to take a look on the last word to see if something has a positive or negative outcome.
To be consistent with "<Subject> <Verb>" I have to do a "NOT" in the negative outcome. that's something that is often overlooked. Thus users may not see the error.

So i chose a 'bad' translation for a better understandability.

What's your view on this ?

**Comment by sluetze** [comment]
> The translation is a bit stiff. There are a few English words still in use although good german word exist also in the IT lingua.
> 
> Roboter-Account -> Roboterkonto
> Secret -> Geheimnis

Hi Vadim,
thanks for the review. I will take a look on the specific suggestions later this day.

"Roboterkonto" is unknown to me in the IT. I only know that word from investments.
Secret/Geheimnis is valid. In my environment no IT person talks about "Geheimnis" we often use secret instead or other english terms. that's a common problem in german "IT-Language".

I'm open to translate all Secrets to Geheimnis though if you think that the meaning would be clear to all.

**Comment by bitsf** [comment]
@sluetze hi, is this PR good enough to merge? 2.2.0 is going to GC.

**Comment by sluetze** [comment]
> @sluetze hi, is this PR good enough to merge? 2.2.0 is going to GC.

@bitsf, yes good enough to merge. I'll try to contact vadim on slack to work on the stiffness in upcoming PRs.

**Comment by AllForNothing** [comment]
@sluetze This line has been updated, I may need to translate it:
image
 Ping me if your PR is ready to merge

**Comment by sluetze** [comment]
@AllForNothing
in 24 hours i'll have a discussion with vadim and maybe we can improve the translation on some parts. If you need to cut out the rc asap, feel free to merge. If you give me 48 hours, I may be able to get the improvements into this PR.

just don't want to block the RC, since the translation is usable.

**Comment by AllForNothing** [comment]
> @AllForNothing
> in 24 hours i'll have a discussion with vadim and maybe we can improve the translation on some parts. If you need to cut out the rc asap, feel free to merge. If you give me 48 hours, I may be able to get the improvements into this PR.
> 
> just don't want to block the RC, since the translation is usable.

@sluetze I'll merge this PR this Friday, then you will have enough time to improve your translation

**Comment by sluetze** [comment]
@Vad1mo , ready for review / comments

**Comment by AllForNothing** [comment]
@sluetze Please rebase your code.

**Comment by sluetze** [comment]
@AllForNothing rebased

**Review by AllForNothing — APPROVED** [review]
lgtm

**Review by Vad1mo — APPROVED** [review]
good work

**Review by danfengliu — APPROVED** [review]

**Pull request merged**